config/environments/development.rb

```diff
@@ -74,17 +74,15 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::FileUpdateChecker
 
-  # config.action_mailer.smtp_settings = {
-  #   address: 'localhost',
-  #   port: '1025',
-  #   domain: 'vsv.io'
+  # config.action_mailer.delivery_method = :mailgun
+  # config.action_mailer.mailgun_settings = {
+  #   api_host: ENV['MAILGUN_API_HOST'],
+  #   api_key: ENV['MAILGUN_API_KEY'],
+  #   domain: ENV['MAILGUN_DOMAIN'],
   # }
-  config.action_mailer.delivery_method = :mailgun
-  config.action_mailer.mailgun_settings = {
-    api_host: ENV['MAILGUN_API_HOST'],
-    api_key: ENV['MAILGUN_API_KEY'],
-    domain: ENV['MAILGUN_DOMAIN'],
-  }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: 'mailcatcher', port: 1025 }
```

Comment:
Could you add a mailcatcher to the docker-compose file, just so that this works out of the box for someone typing `docker-compose up`?

Reply:
good idea